dev/com.ibm.ws.jakartaee.persistence.3.0/bnd.bnd

```diff
@@ -12,7 +12,8 @@
 bVersion=1.0
 
 Bundle-SymbolicName: com.ibm.ws.jakartaee.persistence.3.0; singleton:=true
-#Bundle-Activator: com.ibm.ws.javaee.persistence.internal.HybridPersistenceActivator
+
```

Comment:
Why is this bundle named `com.ibm.ws.jakartaee.persistence.3.0` and not something like `io.openliberty.jakarta.persistence.3.0.internal`?

I am also then confused by the `com.ibm.ws.jakartaee.persistence.api.3.0`. Is that the same as `io.openliberty.jakarta.persistence.3.0`?

Reply:
It's because of the way the JPA APIs had to be structured to support the persistence service.
```
1. com.ibm.websphere.appserver.javax.persistence.base-2.2 -> io.openliberty.jakarta.persistence.base-3.0
2. com.ibm.websphere.appserver.javax.persistence-2.2 -> io.openliberty.jakarta.persistence-3.0
3. com.ibm.websphere.appserver.org.eclipse.persistence-2.7 -> com.ibm.websphere.appserver.org.eclipse.persistence-3.0
4. com.ibm.ws.persistence-1.0 -> io.openliberty.persistenceService-2.0
```

Reply:
3. would be `io.openliberty.org.eclipse.persistence-3.0`?

Reply:
I've renamed the feature. please review.